website/content/docs/ui/custom-messages.mdx

```diff
@@ -0,0 +1,101 @@
+---
+layout: docs
+page_title: Manage custom messages
+description: >-
+  Use custom messages in the Vault UI to share system-wide alerts.
+---
+
+# Manage custom messages in the Vault UI
+
+Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users.
+
+<Tip title="Best practices for UI messages">
+
+1. **Messages are sticky**. Users can only dismiss messages **temporarily**.
+   The message reappears if the user refreshes their browser window or logs out
+   of the current session.
+
+1. **Messages are intrusive**. Limit the number of active messages to minimize
+   the intrusion on users and reduce the chances that they will dismiss the
+   message without reading it.
+
+1. **Messages are inheritable**. Child namespaces inherit all messages created
+   on the parent namespace. Take advantage of inheritance to reach the greatest
+   number of users with the smallest number of messages.
+
+1. **Delete old messages**. Vault supports a maximum of 100 messages per namespace at a
+   time. Practice good message hygiene by regularly deleting expired and outdated
+   messages.
+
+</Tip>
+
+## Before you start
+
+- **You must have Vault 1.16.0 or higher installed**.
+- **You must have the appropriate permissions**:
+  - To **create or edit messages**, you must have the `list` and `create` permissions for the `sys/config/ui/custom-messages` endpoint and the `read` and `update` permissions for the `sys/config/ui/custom-messages/:id` endpoint
+  - To **delete messages**, you must have the `list` permission for the `sys/config/ui/custom-messages` endpoint and the `delete` permission for the `sys/config/ui/custom-messages/:id` endpoint.
+
```

Comment:
@schavis This was changed based on some feedback. Open to more feedback :)

Reply:
Added a suggestion to make the information a little easier to scan.